package.json

```diff
@@ -53,6 +53,7 @@
     "typescript": "^5.0.4"
   },
   "dependencies": {
+    "@biconomy-devx/account-contracts-v2": "^2.0.2",
```

Comment:
This and scw-accounts have circular dependency now. Both paymaster and accounts uses devx version of each other.
Will we publish this package with scw devx package and then update that in scw-account publish that and release new version from here?

Reply:
I see. This is the only circular dependency concern (publishing cycle) or would there b other problems?
I am thinking we could just also rely on submodules. Or only import scw package in paymasters repo. (in scw repo paymaster dependencies if needed can be imported from submodule - or opposite depending on need of dependencies)

Reply:
Yeah the circular dependencies should not be a problem if we can use previous scw-accounts version